.eleventy.js

```diff
@@ -50,6 +50,46 @@ module.exports = function (eleventyConfig) {
   eleventyConfig.addCollection('sortedEntries', collectionApi =>
     getSortedCollection(collectionApi, 'entries'),
   );
+  eleventyConfig.addCollection('indexItems', collectionApi => {
+    const allItems = collectionApi.getFilteredByGlob([
+      'src/entries/*.md',
+      'src/collections/*.md',
+      'src/series/*.md',
+    ]);
+
+    // Get arrays of the entries that belong to collections and series
+    // (use a Set to prevent duplicate values)
+    const entriesThatBelongToCollections = [
+      ...new Set(
+        collectionApi
+          .getFilteredByGlob('src/collections/*.md')
+          .map(collection => collection.data.entries)
+          .flat(),
+      ),
+    ];
+    const entriesThatBelongToSeries = [
+      ...new Set(
+        collectionApi
+          .getFilteredByGlob('src/series/*.md')
+          .map(collection => collection.data.entries)
+          .flat(),
+      ),
+    ];
+
+    // Filter out entries that belong to a collection or series
+    const indexItems = allItems
+      .filter(item => !entriesThatBelongToCollections.includes(item.data.title))
+      .filter(item => !entriesThatBelongToSeries.includes(item.data.title));
```

Comment:
@alecmarcus One thing that concerns me with this approach is the way you're currently adding entries to a collection or a series (title case). If there's a typo or inconsistency with a special character like `&` or `-` in your entries frontmatter array and the corresponding title of the entry, this will break. Of course, you could also make this argument if you used slugs 🤷